### PR TITLE
Add Driver for MPU6050

### PR DESCRIPTION
**Pull request by eyeam3**
This request adds a driver for the IMU-sensor MPU6050 used on the Parrot Bebop 2. As the sensor only supports I2C I had to modify the `framework/include/ImuSensor.hpp` and added a new define which replaces the `SPIDevObj` with an `I2CDevObj`. However, I am a little unsure about the solution.

### CODE REVIEW

File: drivers/mpu6050/MPU6050.hpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+#pragma once
+
+#define __IMU_USE_I2C
```
Comment (julianoes) [inline drivers/mpu6050/MPU6050.hpp]: I think this solution is quite alright. Could you also just use `#if defined(__BEBOP)`?

**Comment by julianoes** [comment]
Reviewed and built/regression tested for Snappy. Seems all good apart from the comment above.

**Comment by eyeam3** [comment]
I replaced the `__IMU_USE_I2C` define and building the driver within the PX4/Firmware works fine. However, if you build the DF standalone then the build fails, as it tries to build an IMU sensor with SPI and one with I2C. Using `__IMU_USE_I2C` prevented that error, as the specific driver defined which version to use.

**Comment by julianoes** [comment]
@eyeam3 I see. Ok let's leave it at that for now then. Thanks.

**Comment by julianoes** [comment]
Rebased and applied, thanks!

**Pull request closed without merging**